src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -424,7 +424,8 @@ private async Task<bool> PushPackageToServer(string source,
             bool showPushCommandPackagePushed = true;
             if (warnForHttpSources && serviceEndpointUrl.Scheme == Uri.UriSchemeHttp && !allowInsecureConnections)
```

Comment:
Should we rename the parameter `warnForHttpSources` now that we are erroring on this condition?